setup.sh

```diff
@@ -52,3 +52,9 @@ sudo rm /var/swap
 # speed up boot
 sudo apt purge exim4-* nfs-common triggerhappy
 sudo apt --purge autoremove
+
+
+# ensure we don't override kernel option for 'ondemand' frequency
+# governor
+sudo systemctl disable loadcpufreq.service
```

Comment:
I'm not seeing this service on my image (190303) so I don't think it needs to be disabled?

Reply:
ah must be a side-effect of the fact I installed cpufreq-utils rather than pranging /proc directly